packages/material-ui/src/Box/Box.test.js

```diff
@@ -22,142 +23,29 @@ describe('<Box />', () => {
     refInstanceof: window.HTMLDivElement,
   }));
 
-  const testChildren = (
-    <div data-testid="child" className="unique">
-      Hello World
-    </div>
-  );
-
-  it('does not forward style props as DOM attributes', () => {
-    const elementRef = React.createRef();
-    render(
-      <Box
-        color="primary.main"
-        fontFamily="Comic Sans"
-        fontSize={{ xs: 'h6.fontSize', sm: 'h4.fontSize', md: 'h3.fontSize' }}
-        ref={elementRef}
-      />,
-    );
-
-    const { current: element } = elementRef;
-    expect(element).not.to.have.attribute('color');
-    expect(element).not.to.have.attribute('font-family');
-    expect(element).not.to.have.attribute('font-size');
-  });
-
-  it('renders children and box content', () => {
-    const { container, getByTestId } = render(
-      <Box component="span" sx={{ m: 1 }}>
-        {testChildren}
-      </Box>,
-    );
-    expect(container.firstChild).contain(getByTestId('child'));
-    expect(container.querySelectorAll('span').length).to.equal(1);
-  });
-
-  it('respect properties order when generating the CSS', function test() {
+  it('respects theme from context', function test() {
```

Comment:
I am testing in the core only if the theme from the context is considered. The other tests were moved to the system.